inventory/internal/store/conversions.go

```diff
@@ -811,3 +811,20 @@ func entIPAddressResourceToProtoIPAddressResource(ipaddress *ent.IPAddressResour
 
 	return protoIPAddress
 }
+
+func entCustomConfigResourceToProtoCustomConfigResource(
+	customconfig *ent.CustomConfigResource,
+) *computev1.CustomConfigResource {
+	if customconfig == nil {
+		return nil
+	}
+	protoLocalAccount := &computev1.CustomConfigResource{
+		ResourceId:  customconfig.ResourceID,
+		Name:        customconfig.Name,
+		Description: customconfig.Description,
+		Config:      customconfig.Config,
+		TenantId:    customconfig.TenantID,
+		CreatedAt:   customconfig.CreatedAt,
+	}
```

Comment:
What about the edge? If we do eager loading (see other comment), we must add it here.